components/console_simple_init/README.md

```diff
@@ -32,6 +32,13 @@ It also provides an api to register an user provided command.
     // This allows you to execute the do_user_cmd function when the "user" command is invoked
     ESP_ERROR_CHECK(console_cmd_user_register("user", do_user_cmd));
 
+    // Register any other plugin command added to your project
+    ESP_ERROR_CHECK(console_cmd_all_register());
 
     ESP_ERROR_CHECK(console_cmd_start());    // Start console
     ```
+
+### Adding a plugin command:
+To add a plugin command or any component from IDF component manager into your project, simply include an entry within the `idf_component.yml` file.
```

Comment:
```suggestion
`console_simple_init` component provides a function `console_cmd_all_register` which performs automatic registration of all commands linked into the application.

In order for the command to be registered automatically, it has to place the registration function into console_simple_init_desc section.

<Here, either add a code snippet showing how to do this, or link to a piece of code in this repository>
```

Reply:
The `<Here, either add a code snippet showing how to do this, or link to a piece of code in this repository>` was supposed to be a placeholder for you to fill :)

Reply:
I committed the suggestion just for convenience.
I have made the necessary changes.